### PR TITLE
metrics: add grafana scripts

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
cp from https://github.com/pingcap/tidb-ansible/blob/master/scripts/br.json

### CODE REVIEW

File: metrics/grafana/br.json [hunk 1]
```diff
@@ -0,0 +1,1690 @@
...
+      "id": "singlestat",
+      "name": "Singlestat",
+      "version": ""
+    }
+  ],
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "${DS_TEST_CLUSTER}",
```
Comment (glorv) [inline metrics/grafana/br.json]: should be `DS_TEST-CLUSTER`?

Comment (WangXiangUSTC) [reply metrics/grafana/br.json]: fixed in aaf7cbb

**Pull request merged**